docs/AboutUs.md

```diff
@@ -5,19 +5,22 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
-
```

Comment:
Would it be better if you instead write: 
```
You can reach us at any one of the following emails: 
- <your email> ?
```

Reply:
Possible. I will add it